packages/_helpers/definitions/events.ts

```diff
@@ -1,7 +1,11 @@
-  export type ClickEvent = typeof HTMLElement.prototype.onclick;
-  export type FocusEvent = typeof HTMLElement.prototype.onfocus;
-  export type BlurEvent = typeof HTMLElement.prototype.onblur;
-  export type KeyupEvent = typeof HTMLElement.prototype.onkeyup;
+export type ClickHandler = ( event: ClickEvent ) => any;
```

Comment:
We don't have to create `<Native>Event` because they are already within prototype definition `HTMLElement.prototype.<native>`.

`Event` types will be defined only for custom Events.

so it should look like this:

```ts
export type ClickHandler = typeof HTMLElement.prototype.onclick;
export type FocusHandler = typeof HTMLElement.prototype.onfocus;
export type BlurHandler = typeof HTMLElement.prototype.onblur;
export type KeyupHandler = typeof HTMLElement.prototype.onkeyup;

// custom events and handlers
export type CustomChangeHandler<T> = ( event: CustomChangeEvent<T> ) => any;
export type CustomChangeEvent<T> = CustomEvent & { detail: { value: T } };
```